--- a/libs/eurorack/CMakeLists.txt
+++ b/libs/eurorack/CMakeLists.txt
@@ -77,5 +77,8 @@ if(MSVC)
 endif()
 
 target_include_directories(${PROJECT_NAME} PUBLIC ${PROJECT_NAME} special_include)
+set_property(TARGET ${PROJECT_NAME} PROPERTY C_VISIBILITY_PRESET default)
+set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_VISIBILITY_PRESET default)
+set_property(TARGET ${PROJECT_NAME} PROPERTY VISIBILITY_INLINES_HIDDEN OFF)
 add_library(surge::${PROJECT_NAME} ALIAS ${PROJECT_NAME})
 
--- a/libs/libsamplerate/src/CMakeLists.txt
+++ b/libs/libsamplerate/src/CMakeLists.txt
@@ -38,6 +38,10 @@ add_library(samplerate
 # ALIAS to use if libsamplerate is included from other project with add_subdirectory()
 add_library(SampleRate::samplerate ALIAS samplerate)
 
+set_property(TARGET samplerate PROPERTY C_VISIBILITY_PRESET default)
+set_property(TARGET samplerate PROPERTY CXX_VISIBILITY_PRESET default)
+set_property(TARGET samplerate PROPERTY VISIBILITY_INLINES_HIDDEN OFF)
+
 # CMake generates wrong DLL names for MinGW and Cygwin, fix it
 if(BUILD_SHARED_LIBS AND WIN32)
   if(LIBSAMPLERATE_COMPATIBLE_NAME)
