--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -80,9 +80,9 @@ endif()
     $<$<NOT:$<OR:$<BOOL:${WIN32}>,$<BOOL:${SURGE_SKIP_WERROR}>>>:-Werror>
 
     # PE/COFF doesn't support visibility
-    $<$<NOT:$<BOOL:${WIN32}>>:-fvisibility=hidden>
+    #$<$<NOT:$<BOOL:${WIN32}>>:-fvisibility=hidden>
     # Inlines visibility is only relevant with C++
-    $<$<AND:$<NOT:$<BOOL:${WIN32}>>,$<COMPILE_LANGUAGE:CXX>>:-fvisibility-inlines-hidden>
+    #$<$<AND:$<NOT:$<BOOL:${WIN32}>>,$<COMPILE_LANGUAGE:CXX>>:-fvisibility-inlines-hidden>
 
     # BP note: If you want to turn on llvm/gcc sanitize undo this and the link options below
     $<$<BOOL:${SURGE_SANITIZE}>:-fsanitize=address>
